src/views/EditPackagingModal.vue

```diff
@@ -54,24 +54,24 @@
 
         <div class="product-metadata">
           <ion-item lines="none">   
-            <ion-label>Select box</ion-label>
+            <ion-label>{{ $t("Select box") }}</ion-label>
             <ion-select>
-              <ion-select-option>Box A  Type3</ion-select-option>
-              <ion-select-option>Box B  Type2</ion-select-option>
+              <ion-select-option>{{ $t("Box A") }} {{ $t("Type 3") }}</ion-select-option>
```

Comment:
I am not sure if this is to be internationalized. I think these values will dynamic. Please verify.

Reply:
Check for everywhere we either taking inputs or showing info for boxes

Reply:
this will be dynamic, no need to internationalize

Reply:
Ok sir